docs/getting-started/local-setup/installation.md

```diff
@@ -20,6 +20,7 @@ to run on your machine.
 ```
 export GOPATH=$HOME/go
 export PATH=$GOPATH/bin:$PATH
+go env -w GOBIN=$GOPATH/bin
```

Comment:
GOPATH/bin is still the default, also on 1.23: https://pkg.go.dev/cmd/go#hdr-Compile_and_install_packages_and_dependencies

Can you show us any errors that occurred / reference to this being required elsewhere?